resource/src/main/java/com/epam/learn/resource/kafka/KafkaTopicListener.java

```diff
@@ -0,0 +1,23 @@
+package com.epam.learn.resource.kafka;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.kafka.annotation.KafkaListener;
+import org.springframework.stereotype.Component;
+
+@Slf4j
+@Component
+public class KafkaTopicListener {
```

Comment:
The name is very generic. Be more specific..

Reply:
Renamed